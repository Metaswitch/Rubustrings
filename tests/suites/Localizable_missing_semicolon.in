/* Just a random string with a wrong format */
"Any String" = "Any String"
