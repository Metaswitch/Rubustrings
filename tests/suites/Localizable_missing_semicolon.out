[34mProcessing file: "tests/suites/Localizable_missing_semicolon.in"
[0m
[31mâœ˜ Error, invalid format: "Any String" = "Any String"
[0m[31m[1m
Result: âœ˜ Some errors detected[0m[0m
